### PR TITLE
Heroku to render

### PR DESCRIPTION
**Pull request by shareoc**

### CODE REVIEW

File: src/docs/ftw-hosting/how-to-deploy-ftw-to-production/index.md [hunk 1]
```diff
@@ -9,280 +9,39 @@ ingress:
...
-  Possible values: true/false
-
-> **NOTE:** If you change these variables, you need to deploy the app
-> again.
-
-![Heroku settings](./heroku-config-vars.png)
-
-### Set up domains and certificates
-
-Heroku manages SSL certificates automatically for new applications. You
```
Comment (SariSaar) [inline src/docs/ftw-hosting/how-to-deploy-ftw-to-production/index.md:257]: The Render tutorial doesn't contain any info on domain and certificate setup => we need to reference these somewhere (e.g. https://render.com/docs/tls and https://render.com/docs/custom-domains)

File: src/docs/ftw-hosting/how-to-deploy-ftw-to-production/index.md [hunk 2]
```diff
@@ -9,280 +9,39 @@ ingress:
 published: true
 ---
 
-## Getting started
```
Comment (SariSaar) [inline src/docs/ftw-hosting/how-to-deploy-ftw-to-production/index.md:12]: The old "Getting started" section can maybe stay as-is, except the Heroku-specific parts. Based on what I understood from Vesa, it's intended for a generic production deployment instruction anyway so we can maybe use it as a base

File: src/docs/ftw-hosting/how-to-deploy-ftw-to-production/index.md [hunk 3]
```diff
@@ -9,80 +9,125 @@ ingress:
...
+development team deems a feature production-ready, i.e. fit for a live
+audience, they can deploy it to production. A workflow like this helps
+prevent bugs and unfinished code from being released to your users.
+
+Sharetribe Flex offers
+[three different environment types](https://www.sharetribe.com/docs/concepts/flex-environments/#environment-types).
+You should connect your client application with the corresponding
+marketplace environment, i.e. your client environment intended for
+testing should use environment variables that point to your test
+environment in Flex-
```
Comment (SariSaar) [inline src/docs/ftw-hosting/how-to-deploy-ftw-to-production/index.md:33]: trailing dash?

File: src/docs/howto-users-and-authentication/setup-oidc-proxy/index.md [hunk 4]
```diff
@@ -235,9 +235,10 @@ The RSA key pair we created in the previous section
 The keys are multi-line strings but Heroku is fine with that so you can
 paste the keys in config vars as they are.
 
-In case your FTW runtime environment requires to declare environment
-variables in a file, wrap the keys with `"`s, escape line breaks with
-`\n`s and join the lines to a single line.
+> If you are using Render or some other environment that requires you to
+> declare environment variables through a file, wrap the RSA keys with
+> quotation marks `"` and escape line breaks with the newline character
+> `/n`. Make sure that the RSA key is defined on a single line.
```
Comment (SariSaar) [inline src/docs/howto-users-and-authentication/setup-oidc-proxy/index.md:241]:
```suggestion
> `\n`. Make sure that the RSA key is defined on a single line.
```

**Review by SariSaar — APPROVED** [review]
A few typo fixes but otherwise looks amazing! 🙌

File: src/docs/ftw-hosting/how-to-deploy-ftw-to-production/index.md (hunk 3 above)
Comment (shareoc) [inline src/docs/ftw-hosting/how-to-deploy-ftw-to-production/index.md:33]:
```suggestion
environment in Flex.
```

**Pull request merged**